doc/source/release-notes/Percona-Server-8.0.13-4.rst

```diff
@@ -0,0 +1,50 @@
+.. rn:: 8.0.13-4
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+|Percona| announces the release of |Percona Server| |release| on
+|date| (downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from
+the `Percona Software Repositories
+<https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+This release contains a fix for a critical bug that prevented |Percona
+Server| 5.7.24-26 from being upgraded to version 8.0.13-3. |Percona
```

Comment:
prevented from being upgrade if there were more than around 1000 tables, or if the maximum allocated InnoDB table ID was higher than around 1000. @satya-bodapati might have exact wording

Reply:
1022 is exact InnoDB table_id. around 1000 is fine. So above wording by @laurynas-biveinis is OK